packages/@tinacms/cli/src/cmds/compile/index.ts

```diff
@@ -226,18 +226,6 @@ export const compileFile = async (
     )
   }
 
-  let schemaExists = true
-  try {
-    getPath({
-      projectDir: tinaPath,
-      filename: fileName,
-      allowedTypes: ['js', 'jsx', 'tsx', 'ts'],
-      errorMessage: `Must provide a ${fileName}.{js,jsx,tsx,ts}`,
-    })
-  } catch {
-    // getSchemaPath will throw an error if it is not found
-    schemaExists = false
```

Comment:
Looks like this needed to be wrapped in a try/catch before. Do you know if that's still the case? @Phoenix-Alpha

Reply:
Actually, `schemaExisits` is not used anymore after this assignment and this try catch block does nothing, I think.
Also, similar file existing check is done in the next try catch block.

Reply:
Are you sure? It doesn't look like they're being wrapped in a try/catch 
https://github.com/tinacms/tinacms/blob/660dc62fbbdb25647d899002b9b6106682190b24/packages/%40tinacms/cli/src/cmds/compile/index.ts#L296-L305

Reply:
For this, I am not sure when it throws an exception as it will return false if a file does not exist.
However, I will wrap with a try/catch block if you want. Please let me know your opinion.

Reply:
Ok cool, yeah just returning true/false seems like the ideal behaviour, so that's fine by me as-is in that case!